### PR TITLE
DOCSP-39938-libmongocrypt-bindings

### PR DESCRIPTION
**Pull request by gmiller-mdb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-39938>

### Staging Links

<li><a href=https://deploy-preview-366--mongodb-docs-csharp.netlify.app/fundamentals/encrypt-fields>fundamentals/encrypt-fields</a></li>

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Comment by gmiller-mdb** [comment]
I am unsure how to made the 'libmongocrypt' heading smaller? I've tried various RST styles and it hasn't been a problem before.

File: source/fundamentals/encrypt-fields.txt [hunk 1]
```diff
@@ -3,3 +3,43 @@
 .. sharedinclude:: dbx/encrypt-fields.rst
 
    .. replacement:: driver-specific-content
+
+Libmongocrypt
```
Comment (adelinowona) [inline source/fundamentals/encrypt-fields.txt:7]: You probably don't need the title anymore

**Review by adelinowona — CHANGES_REQUESTED** [review]

File: source/fundamentals/encrypt-fields.txt [hunk 2]
```diff
@@ -3,3 +3,14 @@
 .. sharedinclude:: dbx/encrypt-fields.rst
 
    .. replacement:: driver-specific-content
+
+To use CSFLE with the driver, you need to have the MongoDB.Driver.Encryption
```
Comment (adelinowona) [inline source/fundamentals/encrypt-fields.txt:7]: Could you hyperlink Mongo.Driver.Encryption to its nuget page? Should be this https://www.nuget.org/packages/MongoDB.Driver.Encryption

File: source/fundamentals/encrypt-fields.txt [hunk 3]
```diff
@@ -3,3 +3,14 @@
...
+
+To use CSFLE with the driver, you need to have the MongoDB.Driver.Encryption 
+nuget package. This package bundles libmongocrypt (library that manages encryption)
+libraries for various operating systems. However, on Linux systems, the bundled 
+libmongocrypt library does not use the system's native encryption providers due 
+to OpenSSL versioning issues. For high-performance applications on Linux, we 
+recommend manually installing the libmongocrypt library and specifying its path 
+to the driver using the LIBMONGOCRYPT_PATH environment variable.
+
+You can find the instructions for downloading a pre-built binary libmongocrypt
```
Comment (adelinowona) [inline source/fundamentals/encrypt-fields.txt:15]: downloading "a libmongocrypt library"

File: source/fundamentals/encrypt-fields.txt [hunk 4]
```diff
@@ -3,3 +3,14 @@
 .. sharedinclude:: dbx/encrypt-fields.rst
 
    .. replacement:: driver-specific-content
+
+To use CSFLE with the driver, you need to have the MongoDB.Driver.Encryption 
+nuget package. This package bundles libmongocrypt (library that manages encryption)
+libraries for various operating systems. However, on Linux systems, the bundled 
+libmongocrypt library does not use the system's native encryption providers due 
+to OpenSSL versioning issues. For high-performance applications on Linux, we 
+recommend manually installing the libmongocrypt library and specifying its path
```
Comment (adelinowona) [inline source/fundamentals/encrypt-fields.txt:12]: replace path with "absolute path"

**Review by adelinowona — CHANGES_REQUESTED** [review]
few more minor comments

**Review by adelinowona — APPROVED** [review]
LGTM!

**Comment by adelinowona** [comment]
Also, as a note, there's a typo in your PR title. bindindings -> bindings

**Comment by gmiller-mdb** [comment]
hi @adelinowona , talked with my team and we decided to move this information into the server docs proper. thanks for all of your help!

**Pull request closed without merging**